src/broadcast.jl

```diff
@@ -8,199 +9,143 @@ else
     const broadcast_indices = broadcast_shape
 end
 
-if VERSION >= v"0.5.0-dev+5189"
-    _to_shape(dims::Base.DimsOrInds) = map(_to_shape, dims)
-    _to_shape(r::Base.OneTo) = Int(last(r))
+if !isdefined(Base.Broadcast, :ftype) # Julia < 0.6
+    ftype(f, A) = typeof(f)
+    ftype(f, A...) = typeof(a -> f(a...))
+    ftype(T::DataType, A) = Type{T}
+    ftype(T::DataType, A...) = Type{T}
```

Comment:
The changes in https://github.com/JuliaLang/julia/pull/19421 might be relevant here. Best!